src/openrct2/object/ObjectRepository.cpp

```diff
@@ -68,13 +68,15 @@ struct ObjectEntryEqual
     }
 };
 
+using ObjectIdentifierMap = std::unordered_map<
+    std::string, size_t, std::hash<std::string_view>, std::equal_to<std::string_view>>;
```

Comment:
You should be able to omit the last two template parameter, unordered-map has those as defaults
```class _Hasher = hash<_Kty>, class _Keyeq = equal_to<_Kty```

Reply:
So this line should read `using ObjectIdentifierMap = std::unordered_map<std::string, size_t>`?

Reply:
Yeah